src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -602,18 +602,16 @@ private static int calculateTagColumn(Matcher javadocTagMatcher,
         final List<JavadocTag> tags = new ArrayList<>();
         final String param1 = argMultilineStart.group(1);
         final String param2 = argMultilineStart.group(2);
-        int remIndex = lineIndex + 1;
-        while (remIndex < lines.length) {
+        for (int remIndex = lineIndex + 1; ; remIndex++) {
```

Comment:
while loop was changed to for loop, which is not really a change.
Condition was removed as we always exit the loop before reaching the end. If this was not the case, we would eventually get an out of index exception.

Reply:
This code was rechanged due to something found in regression.
Not sure why it was missed before.